aioredis/commands/__init__.py

```diff
@@ -52,7 +53,8 @@ class Redis(GenericCommandsMixin, StringCommandsMixin,
             HashCommandsMixin, TransactionsCommandsMixin,
             SortedSetCommandsMixin, ListCommandsMixin,
             ScriptingCommandsMixin, ServerCommandsMixin,
-            PubSubCommandsMixin, ClusterCommandsMixin):
+            PubSubCommandsMixin, ClusterCommandsMixin,
+            SentinelCommandsMixin):
```

Comment:
I don't think that `SentinelCommandsMixin` should be, well, mixed with other commands as Redis Sentinel is not a Redis. It support only few redis commands.
So I think it would be better to make separate `RedisSentinel` class and combine needed mixins